### DISCUSSION TITLE
Master Unlocker V1.78 Rev 9

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#84] [Announcements]
### Changelog
  -  Contains all updates based on the donor version.
   - Some features are only available in the donor version.

<hr /><em>This discussion was created from the release <a href='https://github.com/jhowkNx/Master-Unlocker/releases/tag/1.78-r9'>Master Unlocker V1.78 Rev 9</a>.</em>

**Comment by Newidxe** [comment]
How to convert a file from an archive to lua?

**Comment by ITZuU666** [comment]
It doesn't work

**Comment by identitystealer** [comment]
doesn't work it says this is not compatible i'm on gta v 1.70

**Comment by identitystealer** [comment]
when the hell are you gon update ts it's outdated

**Comment by Farhan-Ihsam** [comment]
Can I grind gta cash safely ? 
If Yes then what amount of cash should I grind daily and by using which feature ?
